c++/src/kj/exception.c++

```diff
@@ -19,19 +19,19 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
+#include <stdlib.h>
```

Comment:
Is it necessary to move this include up? I prefer to have the corresponding header file be the first include, as this ensures that the header itself includes all of its dependencies.